src/docs/product/issues/issue-details/performance-issues/n-one-api-calls.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: "N+1 API Calls"
+sidebar_order: 15
+description: "Learn more about N+1 API calls and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+_N+1 API Calls_ are a performance problem in which an application makes many simultaneous network requests to fetch a given type of resource. For example, an application might make one network request for each item in a list, instead of fetching all the information in a combined network call. Each call to the server incurs some performance overhead, and if the number of calls is high this can lead to severe performance problems. This problems commonly occurs when rendering lists or grids of items, where each item needs to fetch additional information from the server.
+
+## Detection Criteria
+
+The N+1 API Calls detector looks for groups of simultaneous network calls to the same URL. It uses has the following criteria for detection:
+
+- applies to `GET` requests only
+- a minimum of 10 network calls need to happen at the same time
+- the network calls must be a minimum of 50s in duration, and within 5ms of each other
+- the calls must not be to a GraphQL endpoint, or to `_next/data`
+- the calls must not be fetching a static resource (e.g., `.png` files)
+
+## Span Evidence
+
+The evidence for an N+1 queries problem has three main aspects:
```

Comment:
```suggestion
You can identify N+1 problems by three main aspects in the "Span Evidence" section:
```

Reply:
🤔 this section of the doc is trying to explain the contents of the "Span Evidence" section (as in, here's what you get to help you solve this issue), does it make sense to say that this is how they're identified?

Reply:
I like that! Reference to "root cause" helps here, I think!

Reply:
Took a shot, let me know if it's decent